libvast/src/format/json.cpp

```diff
@@ -218,14 +219,19 @@ caf::error add(table_slice_builder& builder, const vast::json::object& xs,
       continue;
     }
     auto x = caf::visit(convert{}, *i, field.type);
-    if (!x)
-      return make_error(ec::convert_error, x.error().context(),
-                        "could not convert", field.name, ":", to_string(*i));
+    if (!x) {
+      if (!err)
+        err = make_error(ec::convert_error);
+      err.context() += x.error().context();
+      err.context() += caf::make_message("could not convert", field.name, ":",
+                                         to_string(*i));
```

Comment:
Why do you use `+=` twice?
Can you give an example of how a "multi-field" error would look like?

Reply:
This is an example with 2 conversion errors in one line:
```
vast.format.json.reader failed to convert value(s) in line 1 : !! convert_error: "long%20double" "to" "vast.string_type" "pcap_cnt" "is" "83" ";" "long%20double" "to" "vast.string_type" "src_port" "is" "1181" ";"
```